plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -794,10 +794,11 @@ public void testSchemaAuthorizationForRole()
         // make sure role-grants only work on existing roles
         assertQueryFails(admin, "ALTER SCHEMA test_schema_authorization_role SET AUTHORIZATION ROLE nonexisting_role", ".*?Role 'nonexisting_role' does not exist in catalog 'hive'");
 
-        assertUpdate(admin, "CREATE ROLE authorized_users IN hive");
-        assertUpdate(admin, "GRANT authorized_users TO user IN hive");
+        String role = "authorized_users" + randomNameSuffix();
+        assertUpdate(admin, "CREATE ROLE " + role + " IN hive");
```

Comment:
it doesn't have to be random.
it's enough that it's not reused between this test and testCreateSchemaWithAuthorizationForRole

Reply:
I know, but random suffix is safer to reduce copy-paste mistake.